src/state/updater.tsx

```diff
@@ -150,23 +164,25 @@ const PricesUpdater = () => {
   }, [])
 
   // TODO: Replace sushi oracle with chainlink
-  const fetchTokenPrices = useCallback(async (provider: Web3Provider) => {
+  const fetchTokenPrices = useCallback(async () => {
     try {
-      const callParams = {
-        abi: OracleInterface,
-        address: ORACLE_ADDRESS[CHAIN_ID],
-        method: 'getPriceUsdc',
-      }
+      const tokenAddresses = [
+        RSR_ADDRESS[ChainId.Mainnet],
+        WETH_ADDRESS[ChainId.Mainnet],
+        WBTC_ADDRESS[ChainId.Mainnet],
+        EURT_ADDRESS[ChainId.Mainnet],
+      ]
+
+      const joinedAddresses = tokenAddresses.join('%2C')
+
+      const res = await fetch(
```

Comment:
Use swr https://swr.vercel.app/es-ES which is already a dependency for fetching this and take advantage of the cache